### DISCUSSION TITLE
$home_net ignored ?

### DISCUSSION THREAD

**Discussion by capitangiaco** [Security-Onion-Solutions/securityonion#6621] [Unsupported Versions] [answered]
I receive a lot of 'ET POLICY DNS Update From External net' with source IP 172.16.0.250 0 and destination IP 172.16.0.171 (both are domain controllers, the second one also DNS server):

{"timestamp":"2021-12-17T11:28:56.725040+0000","flow_id":61160140836912,"in_iface":"bond0","event_type":"alert","src_ip":"172.16.0.250","src_port":53336,"dest_ip":"172.16.0.171","dest_port":53,"proto":"UDP","community_id":"1:nOJbwxKpSIyXN6irUYnx423LhdA=","alert":{"action":"allowed","gid":1,"signature_id":2009702,"rev":5,"signature":"ET POLICY DNS Update From External net","category":"Potential Corporate Privacy Violation","severity":1,"metadata":{"created_at":["2010_07_30"],"updated_at":["2010_07_30"]},"rule":"alert udp $EXTERNAL_NET any -> $HOME_NET 53 (msg:\"ET POLICY DNS Update From External net\"; byte_test:1,!&,128,2; byte_test:1,!&,64,2; byte_test:1,&,32,2; byte_test:1,!&,16,2; byte_test:1,&,8,2; reference:url,doc.emergingthreats.net/2009702; classtype:policy-violation; sid:2009702; rev:5; metadata:created_at 2010_07_30, updated_at 2010_07_30;)"},"app_proto":"dns","payload_printable":"m.(.........._msdcs.*.it......gc._msdcs.*.it........X......","stream":0,"packet":"gBhE7TIoAFBWhDPRCABFAABvJeoAAIARus6sEAD6rBAAq9BYADUAW7tgbfAoAAABAAAAAQAABl9tc2RjcwZjb211bmUGYXJlenpvAml0AAAGAAECZ2MGX21zZGNzBmNvbXVuZQZhcmV6em8CaXQAAAEAAQAAAlgABKwQAPo=","packet_info":{"linktype":1}}

my minion pillar conf:

11 sensor:
     12   interface: 'bond0'
     13   mainip: '192.168.231.182'
     14   mainint: 'ens192'
     15   zeek_lbprocs: 1
     16   suriprocs: 1
     17   manager: 'so-manager'
     18   mtu: 1500
     19   uniqueid: 1623410549
     20   hnsensor: 172.16.0.0/16

(so $home_net is 172.16.0.0/16)
why the connection is considered an update from External Network ?

thanks

Giacomo

**Comment by tomjones1977** [comment, marked as answer]
You need to define the external net - by default it is all IP addresses.  One way to do this would be to set it as `!$HOME_NET` which will make it "any IP other than the HOME_NET range.

https://docs.securityonion.net/en/2.3/suricata.html?highlight=external_net#configuration
